booking.yaml

```diff
@@ -0,0 +1,516 @@
+        lon:
+          $ref: '#/components/schemas/lon'
+    options_meta:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        image:
+          type: string
+          format: url
+        car:
+          type: object
+          properties:
+            passengers:
+              type: integer
+    leg:
+      type: object
+      description: >-
+        A OpenTripPlanner compatible definition of a leg (see OpenTripPlanner
+        docs for reference)
+      additionalProperties: true
+      properties:
+        from:
+          description: The coordinates the TSP should use to resolve leg start location
+          $ref: '#/components/schemas/place'
+        to:
+          description: The coordinates the TSP should use to resolve leg finish location
+          $ref: '#/components/schemas/place'
+        startTime:
+          $ref: '#/components/schemas/time'
+        endTime:
+          $ref: '#/components/schemas/time'
+        mode:
+          $ref: '#/components/schemas/mode'
+        departureDelay:
+          $ref: '#/components/schemas/duration'
+        arrivalDelay:
+          $ref: '#/components/schemas/duration'
+        distance:
+          $ref: '#/components/schemas/distance'
+        fare:
+          $ref: '#/components/schemas/fare'
+        route:
+          type: string
+        routeShortName:
+          type: string
+        routeLongName:
+          type: string
+        agencyId:
+          type: string
+        legGeometry:
+          $ref: '#/components/schemas/legGeometry'
+      required:
+        - from
+        - to
+        - mode
+        - startTime
+        - endTime
+    place:
+      type: object
+      additionalProperties: true
+      properties:
+        name:
+          description: Human readable name of the place
+          type: string
+        stopId:
+          type: string
+        stopCode:
+          type: string
+        lat:
+          $ref: '#/components/schemas/lat'
+        lon:
+          $ref: '#/components/schemas/lon'
+      required:
+        - lon
+        - lat
+    lat:
+      type: number
+      minimum: -90
+      maximum: 90
+    lon:
+      type: number
+      minimum: -180
+      maximum: 180
+    legGeometry:
+      type: object
+      additionalProperties: true
+      properties:
+        points:
+          type: string
+          minLength: 1
+    time:
+      description: >-
+        A UTC timestamp (number of milliseconds in a Date object since January
+        1, 1970, 00:00:00)
+    duration:
+      description: A duration of some time (relative to time) in milliseconds
+      type: integer
+      maximum: 2147483647
+      minimum: 0
+    distance:
+      description: The estimated distance travelled in the leg (in meters)
+      type: integer
+      minimum: 0
+    fare:
+      description: Arbitrary fare data that MaaS will use internally
+      type: object
```

Comment:
Would be good to have a schema or recommendation in place for this.

Reply:
Cool. Can we open an enhancement issue for the fare schema?

Reply:
Yes, let's make this an enhancement issue.

Reply:
Enhancement issue created
https://github.com/maas-alliance/apis/issues/13